README.md

```diff
@@ -9,20 +9,20 @@
 
 ## Features
 
-* Amazing human like typing effects beyond your imagination
-* It does typo, and correct them
-* Small size **[1.5k](https://bundlephobia.com/result?p=shabdawali@2.1.0)**
-* No dependency
-* CPU friendly
+* Amazing human-like typing effects beyond your imagination.
+* It finds typos, and corrects them.
```

Comment:
"It does typo" is correct. Because the library knowingly type wrong words and then correct them to mimic human behavior.